save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/KotlinUtils.kt

```diff
@@ -23,15 +24,41 @@ fun <T : Any> T?.isNotNull(): Boolean = this != null
  * @param action action that should be invoked
  * @return [T] if the result was fetched in [times] attempts, null otherwise
  */
+@Suppress("TooGenericExceptionCaught")
 suspend fun <T : Any> retry(
```

Comment:
may be rename it now, it aim to indicate, that it also collect exceptions now
also, please update `@return` in kdoc

Reply:
oh sure

Reply:
Updated Kdoc, don't want to change the name though